File: back/java/testApp/src/testapp/Payment.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package testapp;

import java.math.BigDecimal;
import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * Платежное поручение
 *
 * @author ili4
 */
public class Payment {
    
    private SimpleDateFormat dateFormatter = new SimpleDateFormat("dd.MM.yyyy");

    /**
     * Дата платежа
     */
    public Date date;

    /**
     * Сумма
     */
    public BigDecimal summa;

    /**
     * Номер платежки
     */
    public String number;

    /**
     * Назначение платежа
     */
    public String description;

    /**
     * Плательщик
     */
    public Counterparty payer;

    /**
     * Получатель
     */
    public Counterparty payee;

    @Override
    public String toString() {
        return "Платёжное поручение\n" + 
                "Дата: " + dateFormatter.format(date) + 
                "\nСумма: " + summa + 
                "\nНомер: " + number + 
                "\nНазначение платежа: " + description + 
                "\nПлательщик:\n" + payer + 
                "\nПолучатель:\n" + payee;
    }

    
}

File: back/php/testApp/payment.php
<?php

class Payment{

    // дата платежа
    public $date;
    // сумма
    public $summa;
    // номер платежки
    public $number;
    // назначение платежа
	public $description;
	// плательщик
	public $payer;	
	// получатель
	public $payee;

    public function __construct(){

        $this->payer = new Counterparty();
        $this->payee = new Counterparty();
    }

    public function toString(){
        return iconv("utf-8", "windows-1251", "<br>Платёжное поручение<br>") . 
             iconv("utf-8", "windows-1251", "<br>Дата: ") . $this->date .
             iconv("utf-8", "windows-1251", "<br>Сумма: ") . $this->summa . 
             iconv("utf-8", "windows-1251", "<br>Номер: ") . $this->number . 
             iconv("utf-8", "windows-1251", "<br>Назначение платежа: ") . $this->description .
             iconv("utf-8", "windows-1251", "<br>Плательщик:<br> ") . $this->payer->toString().
             iconv("utf-8", "windows-1251", "<br>Получатель:<br> ") . $this->payee->toString(). "<br>";
    }
}


?>